### DISCUSSION TITLE
I can't get my website to work properly

### DISCUSSION THREAD

**Discussion by Vivian71349** [community/community#79252] [New to GitHub]
This is my first time using github and I'm trying to upload a website i created using a Bootstrap made template. For some reason though the only thing that shows up on my index html is the text. It isn't formatted correctly and none of the images will show up or anything. My scratch html page does show up though. This is for a school project so I had to create my own page as well as a professional page. I have no idea what to do or how to proceed. Please help!!

 https://github.com/Vivian71349/7Reins.

**Comment by varunpats** [comment]
Hi Vivian, seems like path for the CSS file is incorrect as in the console error is `Failed to load resource: net::ERR_FILE_NOT_FOUND`. Try removing `assets` from path where you import CSS and images. And start path with `./vendor/`. 

If it solves your problem please click `Mark as answer` down below. 
If you need further help, you may reply to this comment.
